backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1714,4 +1714,22 @@ databaseChangeLog:
               now() updated_at,
               migrations_user_id updated_by
             FROM ${database.defaultSchemaName}.person CROSS JOIN migration_user
-            WHERE test_result_delivery IS NOT NULL;
\ No newline at end of file
+            WHERE test_result_delivery IS NOT NULL;
+  - changeSet:
+      id: add-org-verified
+      author: jeremy.a.zitomer@omb.eop.gov
+      comment: Add a required boolean column to the organization table named identity_verified, existing records set to true.
+      changes:
+        - addColumn:
+            tableName: organization
+            columns:
+              - column:
+                  name: identity_verified
+                  type: boolean
+                  remarks: Whether this organization's identity has been verified.
+                  valueBoolean: true
+                  # in the next commit, will change this default value to false
```

Comment:
I think we need to do this all in one step so we keep the database changes completely separate from the other PR.  Should be able to set the default to false and add an update to set the existing rows to true (unless we aren't doing DML in migrations anymore).

Reply:
I would actually argue that the comment should be removed, and we should not plan to change the default value. If the Java persistence code is ignorant of the existence of this field, then we should assume that the API-side changeover to post-verifying IDs hasn't happened yet, and default to `true`; if the persistence code is not ignorant of it, then the Java layer will set this value and the default won't matter.

Reply:
that logic makes sense to me @benwarfield-usds , will go that route